server/src/main/java/org/elasticsearch/health/node/HealthIndicatorDisplayValues.java

```diff
@@ -39,6 +43,14 @@ public static String getTruncatedIndices(Set<String> indices, Metadata clusterMe
         return truncatedIndicesString;
     }
 
+    public static <T> String getSortedUniqueValuesString(Collection<T> values, Predicate<T> predicate, Function<T, String> toString) {
```

Comment:
I think this might've gotten lost in the previous PR but this class and its methods need documentation
https://github.com/elastic/elasticsearch/pull/90262#discussion_r978391466

Reply:
Oh missed it you are right.

Reply:
done